src/ReservoirPair.sol

```diff
@@ -11,5 +11,48 @@ abstract contract ReservoirPair is AssetManagedPair, OracleWriter, ReentrancyGua
         _update(_totalToken0(), _totalToken1(), reserve0, reserve1);
     }
 
-    // todo: may want to implement _update() in this class
+    // force balances to match reserves
+    function skim(address to) external nonReentrant {
+        uint256 _reserve0 = reserve0; // gas savings
+        uint256 _reserve1 = reserve1;
+
+        _returnAndTransfer(token0, to, _totalToken0() - _reserve0, _reserve0, _reserve1);
+        _returnAndTransfer(token1, to, _totalToken1() - _reserve1, _reserve0, _reserve1);
+    }
+
+    function _returnAndTransfer(
+        address aToken, address aDestination, uint256 aAmount, uint256 aReserve0, uint256 aReserve1
+    ) internal {
```

Comment:
I think we should strongly consider adopting forge fmt at some point, we have quite a few styles floating around in the codebase

Reply:
okay honestly I'm only doing this to save $$$ since the audit is calculated by line....

Reply:
oh lol haha, forgot about that